### DISCUSSION TITLE
Possible to run x11?

### DISCUSSION THREAD

**Discussion by rnkn** [isontheline/pro.webssh.net#859] [Q&A] [answered]
Hello, this may be a silly question but is it possible to somehow run x11 on a server and use WebSSH port forwarding to interact with the session from an iOS device?

**Comment by isontheline** [comment, marked as answer]
Hello @rnkn 👋

Unfortunately WebSSH doesn't support the X11 protocol 😢

I'm sorry

**Reply by rnkn** [reply]
No worries. Still the best :)
